modules/serverless-functions-func-yaml-fields.adoc

```diff
@@ -0,0 +1,137 @@
+<4> An environment variable assigned from a key-value pair stored in a config map.
+<5> A set of environment variables imported from key-value pairs of a secret.
+<6> A set of environment variables imported from key-value pairs of a config map.
+
+== volumes
+
+The `volumes` field enables you to mount secrets and config maps as a volume accessible to the function at the specified path. For example:
+
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+volumes:
+- secret: mysecret <1>
+  path: /workspace/secret
+- configMap: myconfigmap <2>
+  path: /workspace/configmap
+----
+<1> The `mysecret` secret is mounted as a volume residing at `/workspace/secret`.
+<2> The `myconfigmap` config map is mounted as a volume residing at `/workspace/configmap`.
+
+== options
+
+The `options` field enables you to modify Knative Service properties for the deployed function, such as autoscaling. If these options are not set, the default ones are used.
+
+These options are available:
+
+* `scale`
+** `min`: The minimum number of replicas. Must be a non-negative integer. The default is 0.
+** `max`: The maximum number of replicas. Must be a non-negative integer. The default is 0, which means no limit.
+** `metric`: Defines which metric type is watched by the Autoscaler. Can be set to `concurrency`, which is the default, or `rps`.
+** `target`: Recommendation for when to scale up based on the number of concurrently incoming requests. The `target` option can be a float value greater than 0.01. The default is 100, unless the `options.resources.limits.concurrency` is set, in which case `target` defaults to its value.
+** `utilization`: Percentage of concurrent requests utilization allowed before scaling up. Can be a float value between 1 and 100. The default is 70.
+* `resources`
+** `requests`
+*** `cpu`: A CPU resource request for the container with deployed function.
+*** `memory`: A memory resource request for the container with deployed function.
+** `limits`
+*** `cpu`: A CPU resource limit for the container with deployed function.
+*** `memory`: A memory resource limit for the container with deployed function.
+*** `concurrency`: Hard Limit of concurrent requests to be processed by a single replica. Can be integer value greater than or equal to 0, default is 0 - meaning no limit.
+
+This is an example configuration of the `scale` options:
+
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+options:
+  scale:
+    min: 0
+    max: 10
+    metric: concurrency
+    target: 75
+    utilization: 75
+  resources:
+    requests:
+      cpu: 100m
+      memory: 128Mi
+    limits:
+      cpu: 1000m
+      memory: 256Mi
+      concurrency: 100
+----
+
+== image
+
+The `image` field sets the image name for your function after it has been built. You can modify `image`. If you do, the next time you run `kn func build` or `kn func deploy`, the function image will be created with the new name.
```

Comment:
just a minor suggestions. Instead of "You can modifiy `image`.", what about "You can modify this field."